nestkernel/node_manager.cpp

```diff
@@ -81,6 +81,13 @@ NodeManager::finalize()
   destruct_nodes_();
 }
 
+void
+NodeManager::change_number_of_threads()
+{
+  // No nodes exist at this point, so nothing to tear down
```

Comment:
How do we know this?

Reply:
Because of the check in `VPManager::set_status()`. I've extended the comment in a885c2e238851b245.